plugins/costinsight/src/hooks/utility.js

```diff
@@ -0,0 +1,146 @@
+
+/*****************DO NOT EDIT CONTENT IN THIS FILE **************/
+
+import { Grid, Card, CardContent, makeStyles } from "@material-ui/core";
+
+/**
+ *  process response */
+
+export function convertSchema(schema, rows) {
```

Comment:
Could we just rename the function to processResponse and remove the comment?

Reply:
we can rename it,  this function Job is really converting Schema data, process response seems like httpResponse. 

if you want to limit the comment, please remove.